verification/curator-service/ui/src/components/App/index.tsx

```diff
@@ -286,6 +290,10 @@ function ProfileMenu(props: { user: User }): JSX.Element {
 
     const classes = menuStyles();
 
+    const releaseNotesUrl =
+        process.env.REACT_APP_RELEASE_NOTES_URL ??
+        `https://github.com/globaldothealth/list/releases/tag/${props.version}`;
```

Comment:
I think actually the second version here, with the format string, is better than using the environment. `process.env` is a node feature and even if you can use it in the browser, that's the environment _of the browser process_ which we don't control. So either get the release notes URL from the backend as we currently do with the version, or build it knowing the version as you currently do: and I prefer the second approach.